_pages/speakers.md

```diff
@@ -13,35 +13,96 @@ permalink: /speakers/
 
 ![](/assets/img/speakers/julia.jpg)
 
+<p class="bio">
 Julia Evans is a software engineer at Stripe. In her spare time, she runs a programming blog about what she's learning (systems programming, often!) at [https://jvns.ca/](https://jvns.ca/) and publishes zines at [https://jvns.ca/zines](https://jvns.ca/zines). She thinks it's important to understand the fundamentals of how your systems work.
+</p>
 
 ### Safia Abdalla
 
 ![](/assets/img/speakers/safia.jpg)
 
+<p class="bio">
 Safia is the founder of Zarf, a platform for independent writers who produce paid subscription content for their readers.
+</p>
 
 ## Invited lightning talk speakers
 
 ### Dinah Davis
 
 ![](/assets/img/speakers/dinah.jpg)
 
+<p class="bio">
 Dinah Davis is the Director of R&D at Arctic Wolf Networks, responsible for the development of the AWN CyberSOC product. Dinah holds an M. Math in cryptography from the University of Waterloo. She has over 16 years of experience in the tech industry with time spent as a Security Software Developer at the Communications Security Establishment Canada and BlackBerry, and as a manager at TrustWave and D2l. Dinah is also the founder of Code Like a Girl, a publication whose goal it is to change society's perceptions of how women are viewed in technology, which has over 25 000 subscribers and a vibrant online community.
+</p>
 
 ### Josh Bowman-Matthews
 
 ![](/assets/img/speakers/josh.jpg)
 
+<p class="bio">
 Josh Bowman-Matthews builds web browsers and sustainable communities for a living. He gets excited about demolishing barriers to participating in open source projects. He also sings in a barbershop quartet.
+</p>
+
+## Lightning talk speakers
```

Comment:
- Should all be capitalized
- "Lightning talk speakers" sounds really awkward to me. How about just "speakers". Not sure what including the format of the conference adds to it

Reply:
was trying to differentiate from keynotes, but I think it'd be clear enough as 'speakers' so I'll do that

keynotes / invited speakers / speakers

not too weird?